cmd/kubelet/app/options/options.go

```diff
@@ -190,6 +191,10 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 		return fmt.Errorf("the SeccompDefault feature gate must be enabled in order to use the --seccomp-default flag")
 	}
 
+	if f.ContainerRuntime != kubetypes.RemoteContainerRuntime {
+		return fmt.Errorf("unsupported CRI runtime: %q", f.ContainerRuntime)
```

Comment:
```suggestion
		return fmt.Errorf("unsupported CRI runtime: %q, only %q is currently supported", f.ContainerRuntime, kubetypes.RemoteContainerRuntime)
```

this would potentially help folks move forward if they are stuck.